### PR TITLE
Forms Automated Test - LPS-130500 | master

### PR DESCRIPTION
**Pull request by ewertonluna**

### CODE REVIEW

**Comment by brittneyq** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -683,6 +740,7 @@ definition {
 		property portal.acceptance = "false";
 
 		// TODO LPS-130511 CanBeCreatedFromAutosavedDraft pending implementation
+
```
Comment (diogocordeiro) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:743]: Hi @ewertonluna , I believe this line is an error on this test.

**Review by diogocordeiro — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase (hunk 1 above)
Comment (ewertonluna) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:743]: Hi, @diogocordeiro! This new line was added after running `ant format-source-current-branch`. The SF command is adding this extra line. I'll look more into it tomorrow. Thank you

**Comment by brittneyq** [comment]
ci:test:sf

**Comment by brittneyq** [comment]
Hi @ewertonluna, please fix your source formatting.

`  [java] java.lang.Exception: Found 1 formatting issues:
     [java] 1: ./portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase expected:<...ding implementation
     [java] [
     [java] ]	}
     [java] 
     [java] 	@description = ...> but was:<...ding implementation
     [java] []	}
     [java] 
     [java] 	@description = ...>
     [java] `

**Comment by diogocordeiro** [comment]
ci:forward

**Pull request closed without merging**